docs/REST-API/99-Rate-Limiting-23.md

```diff
@@ -0,0 +1,74 @@
+---
+tags: [rest-api]
+---
+
+# REST API Rate Limits
+
+<!-- theme: warning -->
+> ### Early Access
+>
+> This page contains the updates that will be published to our [Rate Limiting](../../docs/REST-API/04-Rate-Limiting.md) documentation page as we start 
+> to roll out the 2023 Rate Limiting upgrades.
+
+## What are the limits?
+
+The PagerDuty REST API uses rate limits in order to provide a consistent experience for all of our lovely customers. The limits depend on the type of authorization token being used. Each PagerDuty App has its own set of limits that are separate from both API keys and other apps.
+
+**Account API Keys**
+
+Each Account API Key is allowed 960 requests per minute.
+
+**User API Keys**
+
+User API Keys act as a PagerDuty User and are limited as such. Each user is allowed 960 requests per minute across all of the user's API keys.
+
+**PagerDuty App - App OAuth Tokens**
+
+When a PagerDuty App [uses an app token](../../docs/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty App. Each PagerDuty App is allowed 960 requests per minute against each PagerDuty Account it is authorized to access.
+
+**PagerDuty App - User OAuth Tokens**
+
+When a PagerDuty App [uses a user token](../../docs/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty User. Each PagerDuty App is allowed 960 requests per minute per User it is authorized to act as.
+
+### Operation specific limits
+In a few exceptional circumstances, individual REST API operations may apply additional limits. If an endpoint applies additional limits; it will be noted in the [REST API](/api-reference/) documentation for that endpoint. The `ratelimit-` response headers described later in this document will provide information to the client about the current status of the limit and when it resets.
```

Comment:
When do we plan to update the REST API Endpoint specific pages to reflect the "Operation Specific limit" ? we should also update those in the same go or list those here.

Reply:
Yes those would be updated in the same go. I will add that into the other PR.